app/models/recovery_code.rb

```diff
@@ -0,0 +1,10 @@
+class RecoveryCode < ApplicationRecord
+  self.ignored_columns = %w(encrypted_code)
```

Comment:
This means active record won't load the encrypted code column from the database. Is this really what we need?